src/pivotal-ui/components/meta/package.json

```diff
@@ -4,5 +4,5 @@
     "@npmcorp/pui-css-bootstrap": "^2.0.1",
     "@npmcorp/pui-css-typography": "^4.0.0"
   },
-  "version": "1.3.3"
+  "version": "1.3.4"
```

Comment:
I'm not quite sure why this is necessary (or what happened to version 1.3.3), but you've got my vote.